R/shiny-options.R

```diff
@@ -51,62 +51,60 @@ getShinyOption <- function(name, default = NULL) {
 #'     two seconds).}
-#'   \item{shiny.deprecation.messages}{This controls whether messages for
+#'   \item{shiny.deprecation.messages (defaults to `TRUE`)}{This controls whether messages for
 #'     deprecated functions in Shiny will be printed. See
 #'     [shinyDeprecated()] for more information.}
-#'   \item{shiny.error}{This can be a function which is called when an error
+#'   \item{shiny.error (defaults to `NULL`)}{This can be a function which is called when an error
 #'     occurs. For example, `options(shiny.error=recover)` will result a
 #'     the debugger prompt when an error occurs.}
-#'   \item{shiny.fullstacktrace}{Controls whether "pretty" or full stack traces
-#'     are dumped to the console when errors occur during Shiny app execution.
-#'     The default is `FALSE` (pretty stack traces).}
-#'   \item{shiny.host}{The IP address that Shiny should listen on. See
+#'   \item{shiny.fullstacktrace (defaults to `FALSE`)}{Controls whether "pretty" (`FALSE`) or full
+#'     stack traces (`TRUE`) are dumped to the console when errors occur during Shiny app execution.
+#'     Pretty stack traces attempt to only show user-supplied code, but this pruning can't always
+#'     be done 100\% correctly.}
+#'   \item{shiny.host (defaults to `"127.0.0.1"`)}{The IP address that Shiny should listen on. See
 #'     [runApp()] for more information.}
-#'   \item{shiny.json.digits}{The number of digits to use when converting
+#'   \item{shiny.json.digits (defaults to `16`)}{The number of digits to use when converting
 #'     numbers to JSON format to send to the client web browser.}
-#'   \item{shiny.launch.browser}{A boolean which controls the default behavior
+#'   \item{shiny.launch.browser (defaults to `interactive()`)}{A boolean which controls the default behavior
 #'     when an app is run. See [runApp()] for more information.}
-#'   \item{shiny.maxRequestSize}{This is a number which specifies the maximum
-#'     web request size, which serves as a size limit for file uploads. If
-#'     unset, the maximum request size defaults to 5MB.}
-#'   \item{shiny.minified}{If this is `TRUE` or unset (the default), then
-#'     Shiny will use minified JavaScript (`shiny.min.js`). If
-#'     `FALSE`, then Shiny will use the un-minified JavaScript
-#'     (`shiny.js`); this can be useful during development.}
-#'   \item{shiny.port}{A port number that Shiny will listen on. See
+#'   \item{shiny.maxRequestSize (defaults to 5MB)}{This is a number which specifies the maximum
+#'     web request size, which serves as a size limit for file uploads.}
+#'   \item{shiny.minified (defaults to `TRUE`)}{By default
+#'     Whether or not to include Shiny's JavaScript as a minified (`shiny.min.js`)
+#'     or un-minified (`shiny.js`) file. The un-minified version is larger,
+#'     but can be helpful for development and debugging.}
+#'   \item{shiny.port (defaults to `httpuv::randomPort()`)}{A port number that Shiny will listen on. See
```

Comment:
Can change this to `randomPort()` if and when #2599 is merged (otherwise, it should say something like `defaults to a random open port`)